umake/frameworks/baseinstaller.py

```diff
@@ -41,6 +41,11 @@
 class BaseInstaller(umake.frameworks.BaseFramework):
 
     DIRECT_COPY_EXT = ['.svg', '.png', '.ico', '.jpg', '.jpeg']
+    # Framework environment variables are added to `~/.profile` which may
+    # require logging back into your session for the changes to be picked up.
+    # Use `POST_INSTALL_WARN` to alert users to this fact, in `post_install`
+    # function. {} in replaced with the framework name at runtime.
+    POST_INSTALL_WARN = "You may need to log back in for your {} installation to work properly"
```

Comment:
I would change the variable name to `RELOGIN_REQUIRE_MSG`.

Also, this is the variable that needs to have = _("…"). So, you can remove it from the other calls, and only have it once for translation here (the other calls isn't useful, as gettext won't be able to pick up the text assignement properly).